src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let costs = 0;
+
+  if (days === 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    costs = 40 * days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
To make this line smaller, you can go from largest to smallest 🙂
` if (days >= 7) {
return totalRent- bigDiscount;
}

if (days >= 3) {
return totalRent - smallDiscount;
}`